lib/assisted_workflow/addons/github.rb

```diff
@@ -22,12 +22,8 @@ def description
       @issue.body.to_s.gsub("\r\n", "\n")
     end
     
-    def other_id
```

Comment:
ah yes, now I remember. I've set this because I would like to use a different user name in branch names. I have a long github username and I'd like to use just 'flavio'. But I see the story in tracker_api does not have an other_id. I don't see any problem on using github username.